README.md

```diff
@@ -1,8 +1,150 @@
+<div align="center">
+  <a href="https://novu.co" target="_blank">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/2233092/213641039-220ac15f-f367-4d13-9eaf-56e79433b8c1.png">
+    <img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280" alt="Logo"/>
+  </picture>
+  </a>
+</div>
+
 # Novu Java SDK
 
+[![License](https://poser.pugx.org/unicodeveloper/novu/license.svg)](LICENSE.md)
+
 Novu's API exposes the entire Novu features via a standardized programmatic interface. Please refer to the full [documentation](https://docs.novu.co/docs/overview/introduction) to learn more.
 
+## Installation
+
+```xml
+// add dependency
+<dependency>
+    <groupId>co.novu</groupId>
+    <artifactId>novu-java</artifactId>
+    <version>1.0.0</version>
+</dependency>
+```
+Then run `mnv install`.
+
+
+## List of all methods
+
+The client methods map directly to the Novu API endpoints. Here is a list of all the available methods. Check [the API docs](https://docs.novu.co/api/overview) for list of available `methods`.
+
+### Changes
+
+- `changes(query = {})`
+- `countChanges()`
+- `applyBulkChanges()`
+- `applyChange(changeId)`
+
+### Environments
+
+- `currentEnvironment()`
+- `createEnvironment(body)`
+- `environments()`
+- `updateEnvironment(environmentId, body)`
+- `apiKeys()`
+- `regenerateApiKeys()`
+- `updateWidgetSettings(body)`
+
+### Events
+
+- `triggerEvent(body)`
+- `triggerBulkEvent(body)`
+- `broadcastEvent(body)`
+- `cancelTriggeredEvent(transactionId)`
```

Comment:
1. Please can we move this to the top of the methods? This is what most people interact with or do most of the times with our SDKs.
2. Please can we provide actual examples? Just as we have in the [PHP](https://github.com/novuhq/novu-php/blob/master/README.md) & [Ruby SDKs](https://github.com/novuhq/novu-ruby/blob/main/README.md)

You don't have to provide all examples right now but we should have for Events, Subscribers and Topics for now. These three methods should also be at the top because that's what a lot of people interact with.

Reply:
This has been completed